po/vi.po

```diff
@@ -6188,57 +6198,75 @@ msgid ""
 "to [grapheme clusters](https://docs.rs/unicode-segmentation/latest/"
 "unicode_segmentation/struct.Graphemes.html)."
 msgstr ""
+"`String::chars` trả về một con trỏ chạy dọc theo các ký tự của chuỗi. Lưu ý "
+"rằng một `char` có thể khác với một \"ký tự\" thông thường bởi vì một khái "
+"niệm gọi là [grapheme clusters](https://docs.rs/unicode-segmentation/latest/"
+"unicode_segmentation/struct.Graphemes.html)."
 
 #: src/std-types/string.md
 msgid ""
 "When people refer to strings they could either be talking about `&str` or "
 "`String`."
-msgstr ""
+msgstr "Khi nói đến chuỗi, ta có thể sử dụng `&str` hoặc `String` đều được."
 
 #: src/std-types/string.md
 msgid ""
 "When a type implements `Deref<Target = T>`, the compiler will let you "
 "transparently call methods from `T`."
 msgstr ""
+"Khi một kiểu dữ liệu implement `Deref<Target = T>`, trình biên dịch sẽ cho "
```

Comment:
"implement" not translated?

Reply:
this implements is not a built-in in C++ btw. implements (some Vietnamese translation) should be good

Reply:
`Deref<Target = T>` is a trait, so this `implement` should be a technical term in Rust

Reply:
"When a type implements `Deref<Target = T>`". What I understand here is  I think "implements" here is not a technical term in Rust since it has "s" in the word? Are you sure it's not? This is what I got from using a translator:
"Khi một kiểu biến triển khai `Deref<Target = T>`, trình biên dịch sẽ cho phép bạn .." 

The `implement` you referred to is without s. But in that sentence implement also go with "s". So is it a technical term or not?

Reply:
Haha, I agree with you that it's quite confused here. `implement` is being used here with a suffix of `s` because it was "following English grammar rule". However, I can assured you this is a technical term

Reply:
@qu-ngx If you are unsure, perharps this chapter in the Rust book (see [link](https://doc.rust-lang.org/book/ch10-02-traits.html#implementing-a-trait-on-a-type)) can help you make a decision.

Reply:
Ah, I should mention it, `implement` here is actually the expansion of `impl` keyword. That means `Vec implements Deref<Target = T>` is actually the same as `impl Deref<Target = T> for Vec`

Reply:
Yea, you better add it or put it somewhere in the brackets. And also if you don't translate the implements word then better keep the s with it as well so it should be implements instead of implement.